packages/datagateway-dataview/public/datagateway-dataview-settings.example.json

```diff
@@ -46,23 +46,29 @@
     }
   ],
   "routes": [
+    {
+      "section": "Homepage",
+      "link": "/datagateway",
+      "displayName": "DataGateway",
+      "order": 0
+    },
```

Comment:
Homepage needs to be a registered route for at least one plugin for SciGateway to access it. It was decided to use dataview for this since there is currently no implementation of DataGateway that does not use dataview.

Reply:
The navigation drawer now contains a link that directs to the homepage when clicked on it. I think that it would be better not to have this. I already did something similar for the admin plugins as part of the [Display admin functionality from plugins](https://github.com/ral-facilities/scigateway/pull/679) PR in the `navigationDrawer.component.tsx` file,

Reply:
This is something I'll fix on the SciGateway side of things

Reply:
This has now been fixed in[ this commit](https://github.com/ral-facilities/scigateway/pull/704/commits/309ad488cfc0da6189170d013b3e8f85d70eaa45).